pkg/cli/admin/release/config.go

```diff
@@ -0,0 +1,58 @@
+package release
+
+import (
+	"encoding/json"
+
+	configv1 "github.com/openshift/api/config/v1"
+
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/serializer"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+)
+
+var (
+	configScheme = runtime.NewScheme()
+	configCodecs = serializer.NewCodecFactory(configScheme)
+)
+
+func init() {
+	utilruntime.Must(configv1.AddToScheme(configScheme))
+}
+
+func ReadFeatureGateV1(objBytes []byte) (*configv1.FeatureGate, error) {
+	requiredObj, err := runtime.Decode(configCodecs.UniversalDecoder(configv1.SchemeGroupVersion), objBytes)
+	if err != nil {
+		return nil, err
+	}
+
+	return requiredObj.(*configv1.FeatureGate), nil
+}
+
+func ReadFeatureGateV1OrDie(objBytes []byte) *configv1.FeatureGate {
+	requiredObj, err := ReadFeatureGateV1(objBytes)
+	if err != nil {
+		panic(err)
+	}
+
+	return requiredObj
+}
+func WriteFeatureGateV1(obj *configv1.FeatureGate) (string, error) {
```

Comment:
we don't use/need WriteFeatureGateV1 functions, are we having these for consistency?

Reply:
> we don't use/need WriteFeatureGateV1 functions, are we having these for consistency?

Yes.  Preferred to add here: https://github.com/openshift/library-go/pull/1659